dll/win32/advapi32/wine/security.c

```diff
@@ -1,13 +1,11 @@
 /*
- * COPYRIGHT:       See COPYING in the top level directory
+ * PROJECT:     ReactOS system libraries
+ * LICENSE:     See COPYING in the top level directory
  * WINE COPYRIGHT:
  * Copyright 1999, 2000 Juergen Schmied <juergen.schmied@debitel.net>
  * Copyright 2003 CodeWeavers Inc. (Ulrich Czekalla)
  * Copyright 2006 Robert Reif
  * Copyright 2006 Hervé Poussineau
- *
- * PROJECT:         ReactOS system libraries
- * FILE:            dll/win32/advapi32/wine/security.c
```

Comment:
"Hacking" AdvApi32, with no actual details, seems unexpected to me.
Do we know if the TaskMgr failure is related to a specific process or conditions, which we hopefully could check and skip there (or maybe here)?

Reply:
See https://jira.reactos.org/browse/CORE-18521?focusedId=140354&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-140354

Reply:
> See 

I already read that Jira comment. My GitHub question stands.